packages/storybook8/stories/Components/SendBox/RichTextSendBox/snippets/RichTextSendBoxWithInlineImages.snippet.tsx

```diff
@@ -0,0 +1,35 @@
+import { RichTextSendBox, FluentThemeProvider, AttachmentMetadataInProgress } from '@azure/communication-react';
+import React, { useState } from 'react';
+
+export const RichTextSendBoxWithInlineImagesExample: () => JSX.Element = () => {
+  const [inlineImages, setInlineImages] = useState<AttachmentMetadataInProgress[] | undefined>();
+
+  return (
+    <FluentThemeProvider>
+      <div style={{ width: '31.25rem' }}>
+        <RichTextSendBox
+          onSendMessage={async () => {
+            setInlineImages(undefined);
+            return;
+          }}
+          onInsertInlineImage={(image: string, fileName: string) => {
+            const id = inlineImages?.length ? (inlineImages.length + 1).toString() : '1';
+            const newImage = {
+              id,
+              name: fileName,
+              progress: 1,
+              url: image,
+              error: undefined
+            };
+            setInlineImages([...(inlineImages ?? []), newImage]);
+          }}
+          inlineImages={inlineImages}
+          onCancelInlineImageUpload={(imageId: string) => {
+            const filteredInlineImages = inlineImages?.filter((image) => image.id !== imageId);
+            setInlineImages(filteredInlineImages);
+          }}
+        />
+      </div>
+    </FluentThemeProvider>
```

Comment:
We probably need to merge from main and update this snippet page. InlineImage related APIs are outdated.

Reply:
Updated